packages/server/src/routes/members/index.ts

```diff
@@ -210,6 +212,32 @@ async function movePrivateVideoToPublicInviteVideo(
 
   return publicVideoRef;
 }
+/**
+ * Send a push notification. Uses Firebase Cloud Messaging.
+ */
+async function _sendPushNotification(
+  messaging: adminMessaging.Messaging,
+  fcmTokens: CollectionReference,
+  toMemberId: MemberId,
+  title: string,
+  body: string
+) {
+  const fcmTokenData = await fcmTokens.doc(toMemberId).get();
+  const fcmToken = fcmTokenData.exists
+    ? fcmTokenData.get("fcm_token")
+    : undefined;
+
+  if (fcmToken) {
+    await messaging.send({
+      // Note: May need to display an icon for the notification.
```

Comment:
Should we make an issue for this? Or are they notes for you?

Reply:
Mm, I guess I don't consider the issue for notifications completed until this note gets done. Kind of noise to have in the code though. Removed.